projects/packages/post-list/src/class-post-list.php

```diff
@@ -91,6 +91,13 @@ public function enqueue_scripts( $hook ) {
 				'rtl',
 				plugin_dir_url( __DIR__ ) . './src/rtl.css'
 			);
+			wp_enqueue_script(
+				'jetpack_posts_list',
+				plugin_dir_url( __DIR__ ) . './build/index.js',
+				array(),
+				self::PACKAGE_VERSION,
+				true
+			);
```

Comment:
You might consider using `Assets::register_script()` from [automattic/jetpack-assets](https://packagist.org/packages/automattic/jetpack-assets) here. For Webpack-built bundles it handles some stuff you left out here, like reading the dependencies and version-hash from the `build/index.asset.php` file and setting the textdomain for the script. 🙂

An example usage:
https://github.com/Automattic/jetpack/blob/5711116c26c4461c7baede0a3a43703d45448146/projects/packages/my-jetpack/src/class-initializer.php#L218-L227

Reply:
Sorry for the possible naive question, but what is `build/index.asset.php`?